src/engines/aurora/gui.h

```diff
@@ -115,10 +115,14 @@ class GUI : boost::noncopyable {
 	/** Callback that's triggered when a widget was activated. */
 	virtual void callbackActive(Widget &widget);
 
+	void addChild(GUI*);
```

Comment:
because of the event handlers, i will look into it, so that a change can be made inside a callback. That is definitly needed when a event should add a new child GUI.